src/navigation_controller.ts

```diff
@@ -901,6 +927,33 @@ export class NavigationController {
     ContextMenuRegistry.registry.register(deleteItem);
   }
 
+  /**
+   * Register the block copy action as a context menu item on blocks.
+   */
+  protected registerCopyAction() {
+    const copyAction: ContextMenuRegistry.RegistryItem = {
+      displayText: (scope) => {
+        return 'Keyboard Navigation: copy';
+      },
```

Comment:
Curlies and `return` are optional:
```suggestion
      displayText: (scope) => 'Keyboard Navigation: copy',
```